src/relcast.erl

```diff
@@ -620,8 +623,17 @@ stop(Reason, State = #state{module=Module, module_state=ModuleState})->
             ok
     end,
     State1 = maybe_serialize(State),
-    catch rocksdb:transaction_commit(State1#state.transaction),
-    rocksdb:close(State#state.db).
+    %% catch rocksdb:transaction_commit(State1#state.transaction),
+    rocksdb:flush(State#state.db, [{wait, true}, {allow_write_stall, true}]),
+    ok = rocksdb:close(State#state.db),
+    check_stop(State#state.dir).
+
+check_stop(Dir) ->
+    Stop = Dir ++ "/check_stop",
+    ok = file:write_file(Stop, <<"check stop">>, [sync]),
+    ok = file:delete(Stop),
```

Comment:
Is the goal here to serialize in the disk buffer behind the rocksdb close?

Reply:
that was the idea.  I don't think that this is actually necessary.  I was seeing consistent state loss with a 1.5 second settling time after a close, I think that it had something to do with how libp2p was using it, see that PR.